ccmlib/common.py

```diff
@@ -121,10 +121,12 @@ def replaces_or_add_into_file_tail(file, replacement_list):
                     if match:
                         line = replace + "\n"
                         is_line_found = True
-                f_tmp.write(line)
+                if line != "</configuration>\n":
```

Comment:
Instead, could you use a less restrictive check? Such as `if "</configuration" not in line`?